global_uid.gemspec

```diff
@@ -1,4 +1,4 @@
-Gem::Specification.new 'global_uid', '3.7.1' do |s|
+Gem::Specification.new 'global_uid', '4.0.0' do |s|
```

Comment:
How about we release a v4.0.0.beta.1 first? There are a lot of changes, and we probably want to verify that the new version works before making the big release.

Reply:
Fair point, I've already done some testing but I'll release a beta first in case there's something we've missed